src/assets/backArrow.svg

```diff
@@ -1,3 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.004 21.465">
-  <path id="Union_2" data-name="Union 2" d="M10.373,21.587.706,11.921a1,1,0,0,1-.29-.78,1,1,0,0,1,.29-.769L10.373.707a1,1,0,0,1,1.414,1.414L3.609,10.3H25.418a1,1,0,0,1,0,2H3.914l7.872,7.872a1,1,0,0,1-1.414,1.415Z" transform="translate(-0.414 -0.414)" fill="#f1f1f1"/>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
let's don't add explicit sizes to icons that didn't have a size before
it has been discussed [there](https://github.com/aeternity/superhero-ui/pull/893#discussion_r530066146)